assets/style.css

```diff
@@ -0,0 +1,682 @@
+body {
+  overflow-x: hidden;
+  font-family: 'Open Sans', sans-serif;
+}
+
+header {
+  background-color: #fff;
+  height: 6vh;
+}
+
+main {
+  background-color: white;
+  margin-top: 7.6vh;
+  height: 608.8vh;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `background-color: white;` and `background-color: #fff;` properties in entire CSS file, and create just one global for both of them
- [ ] Add your HTML tags


**Example:**

**CSS file**

```CSS
.white-background{
  background-color: #fff;
}
```
**Html file**

```HTML
<div class="white-background">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_